01-syntax/04-high-order-functions/life-expectancy/lifeExpectancy.js

```diff
@@ -1,14 +1,42 @@
+/* global ancestry lifeExpectancy */
+
 function average(array) {
   function plus(a, b) {
     return a + b;
   }
   return array.reduce(plus) / array.length;
 }
 
-console.log(average([2, 2, 2]));
-// → 2
+function groupByCentury(arr) {
```

Comment:
Becuase we don't use any module system here, let's push all functions inside 
`lifeExpectancy` it would be more proper solution from encapsulation point of view